### PR TITLE
Friendlier Danger bot

### PR DESCRIPTION
**Pull request by cpojer**
Danger bot is pretty aggressive. This makes it a bit more friendly and helpful.

## Test Plan
See screenshot (this PR previously had an empty summary):

screen shot 2019-01-23 at 14 23 32

## Changelog
[General] [Changed] Danger bot is now friendlier in the RN repo.

### CODE REVIEW

File: bots/dangerfile.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const includes = require('lodash.includes');
 const {danger, fail, warn} = require('danger');
 
 // Fails if the description is too short.
-if (!danger.github.pr.body || danger.github.pr.body.length < 10) {
+if (!danger.github.pr.body || danger.github.pr.body.length < 50) {
```
Comment (cpojer) [inline bots/dangerfile.js]: The minimum text expected by this file is already ~40 characters, so bumping this.

File: bots/dangerfile.js [hunk 2]
```diff
@@ -47,20 +50,29 @@ const correctlyFormattedChangelog = changelogRegex.test(danger.github.pr.body);
...
+    'format your changes to match this regular expresion: ' +
+    changelogRegex.toString();
   warn(`${title} - <i>${idea}</i>`);
 }
 
 // Tags big PRs
 var bigPRThreshold = 600;
 if (danger.github.pr.additions + danger.github.pr.deletions > bigPRThreshold) {
   const title = ':exclamation: Big PR';
-  const idea = `This PR is extremely unlikely to get reviewed because it touches ${danger
```
Comment (cpojer) [inline bots/dangerfile.js]: `extremely` sounds aggressive.

File: bots/dangerfile.js [hunk 3]
```diff
@@ -47,20 +50,29 @@ const correctlyFormattedChangelog = changelogRegex.test(danger.github.pr.body);
 
 if (!includesChangelog) {
   const title = ':clipboard: Changelog';
-  const idea = 'This PR appears to be missing Changelog.';
+  const idea =
+    'This PR appears to be missing Changelog. ' +
```
Comment (cpojer) [inline bots/dangerfile.js]: I see this a lot and don't even know myself what is expected of me.

File: bots/dangerfile.js [hunk 4]
```diff
@@ -34,7 +34,10 @@ const includesTestPlan =
   danger.github.pr.body.toLowerCase().includes('test plan');
 if (!includesTestPlan) {
   const title = ':clipboard: Test Plan';
-  const idea = 'This PR appears to be missing a Test Plan.';
+  const idea =
+    'This PR appears to be missing a Test Plan. ' +
+    'Please add a section called "test plan" describing ' +
```
Comment (cpojer) [inline bots/dangerfile.js]: Asking nicely.

**Comment by hramos** [comment]
<3

**Pull request closed without merging**